src/Paginator.php

```diff
@@ -661,7 +661,7 @@ public function getIterator()
         try {
             return $this->getCurrentItems();
         } catch (Throwable $e) {
-            throw new Exception\RuntimeException('Error producing an iterator', null, $e);
+            throw new Exception\RuntimeException('Error producing an iterator', 0, $e);
```

Comment:
Note that it is not the only occurrence of null as second parameter. Rest of them would need to be handled as well.

Reply:
Also, it seems to me that unit tests are useless in this case. What do you think?

Reply:
Yeah, I think reducing the psalm baseline suffices: `vendor/bin/psalm --update-baseline` should do that

Reply:
@Xerkus I've looked through classes, this is the only place.

Reply:
@Ocramius the branch has been rebased on top of `2.12.x`.

Reply:
> @Xerkus I've looked through classes, this is the only place.

Indeed. My bad. Used wrong grep regex.